Expand on bug control team, add bug squad (#133)
ubuntu/ubuntu-project-docs: docs/who-makes-ubuntu/specialist-teams/bug-squad.md

(bug-squad)=
# The Bug Squad

The Bug Squad is an essential asset in progressing Ubuntu and its flavours; the
Bug Squad is the first point of contact for bugs filed about Ubuntu.

## Who is in the Bug Squad?

Anyone interested in triaging bugs can join the Bug Squad. The
[members](https://launchpad.net/~bugsquad/+members) are primarily volunteers

> from around the world and they come together with a common cause of making

Reviewer: ```suggestion
from around the world, and they come together with a common cause of making
```

Ubuntu better for everyone.

**No special knowledge or training is required**, just a desire to help!

## What the team does

> What the Bug Squad does is also known as "triaging", which is similar to the
> process of prioritizing patients in a hospital based on the severity of their
> condition.

Reviewer: I know the analogy has basis in a real-world thing, but I'd rather not use hospitals and patients... What about:
```suggestion
What the Bug Squad does is also known as "triaging" -- a process that ensures
bug reports are ranked in terms of importance or priority. 
```

Author: Yeah I removed that in the main triage page (not part of this PR) -- it makes sense to remove it here too.

> This allows developers to spend their time more effectively, and is a great way
> to learn about how Ubuntu works as an operating system and as a community. It’s
> also not uncommon for triagers to discover new and interesting features they
> didn't know about!

Reviewer: ```suggestion
This allows developers to spend their time more effectively and is a great way
to learn how Ubuntu works as an operating system and as a community. It’s
also not uncommon for triagers to discover new and interesting features they
didn't know about.
```

Triaging bugs consists of:

* Responding to new bugs as they are filed
* Ensuring that new bugs have all the necessary information
* Assigning bugs to the proper package
* Confirming bug reports by trying to reproduce them
* Setting the importance of bugs reports ({ref}`bug-control-team`team only)

> * Searching for and marking duplicates in the bug tracking system

Reviewer: ```suggestion
* Searching for and marking duplicates in the bug-tracking system
```

* Sending bugs to their upstream authors, when applicable
* Cross-referencing bugs from other distributions
* Expiring old bugs

Bug triage is an excellent way to start helping out. You get to learn a lot
about Ubuntu, its available packages, its infrastructure, and you get a feel
for the development pulse.

Another big part of what we do is help new members to triage bugs. On average,
over 1,800 bugs are filed every week with more than 500,000 bugs filed so far.
The Bug Squad is always in NEED of more help from the community!

## Contact the squad

* IRC: `#ubuntu-bugs` channel to contact Ubuntu Bug Squad members for help or
  collaboration.

> * Mailing List: sign up to the

Reviewer: ```suggestion
* Mailing List: subscribe to
```

  [`ubuntu-bugsquad`](https://lists.ubuntu.com/mailman/listinfo/ubuntu-bugsquad)
  to contact us if you have any doubt while triaging a bug. 

## Want to join?

> If you want to join the Bug Squad team you need to:

Reviewer: ```suggestion
To join the Bug Squad team:
```

* Sign the {ref}`Ubuntu Code of Conduct <code-of-conduct>`

* Read [the Triage Guide](https://wiki.ubuntu.com/Bugs/Triage)

* Subscribe to the [Bug Squad Mailing List](https://lists.ubuntu.com/mailman/listinfo/ubuntu-bugsquad) 

If you completed the previous steps,
[apply for membership](https://launchpad.net/~bugsquad/+join) on the Bug Squad
team at Launchpad.

When joining the Bug Squad, please keep up-to-date with bug management policies.

> Changes are always announced on the mailing list. For further information feel
> free to subscribe to the wiki changes or check with other members on
> `#ubuntu-bugs` channel on libera.

Reviewer: Subscribing to the wiki should pbbly not be advised anymore.
```suggestion
Changes are always announced on the mailing list. For further information,
check with other members on the `#ubuntu-bugs` channel on Libera.
```

> Now, please read this page for more detailed instructions. 

Reviewer: ```suggestion
Read this page for more detailed instructions. 
```
